docs/sources/visualizations/geomap.md

```diff
@@ -12,6 +12,104 @@ The Geomap panel visualization allows you to view and customize the world map us
 
 {{< figure src="/static/img/docs/geomap-panel/geomap-example-8-1-0.png" max-width="1200px" caption="Geomap panel" >}}
 
+## Map View
+
+The map view controls the initial view of the map when the dashboard loads. 
+
+### Initial View
+
+The initial view configures how the GeoMap panel renders when the panel is first loaded.
+
+* **View** sets the center for the map when the panel first loads.
+* **Latitude** (available when the **View** mode is *Coordinates*)
+* **Longitude** (available when the **View** mode is *Coordinates*)
+* **Zoom** sets the initial zoom level for the GeoMap panel.
+
+## Data layer
+
+The Geomap visualization supports multiple Data Layers. Each data layer determines how you visualize geospatial data on top of the base map.
+
+### Layer Types
+
+There are four layer types to choose from in the Geomap visualization.
+
+* **Marker** renders a marker at each data point.
+* **Heatmap** visualizes a heatmap of the data.
+* **GeoJSON** renders static data from a geojson file.
+* **Icon at last point (alpha)** renders an icon at the last point in a series of points.
+
+### Layer Controls
+
+The layer controls allow you to create layers, change their name, reorder and delete layers.
+
+* **Add layer** creates an additional, configurable data layer for the Geomap visualization. When you add a layer you will be prompted to select a layer type. The layer type can be changed at any point during panel configuration. See the **Layer Types** section above for details on each layer type.
+* The layer controls allow you to rename, delete, and reorder the layers of the panel.
```

Comment:
It may be good to note what layer order means (I'm actually not sure myself), whether that means z-index of how we render the data (i.e. impacts layer visibility) or something else

Reply:
Great call out - it is indeed a relative z-index and the layers which are higher on the list will display over the lower layers - so while it works as expected this is a great call out to clarify. Thanks!